packages/api/src/routers/api/dashboards.ts

```diff
@@ -108,12 +108,13 @@ router.post(
       }
 
       const { name, charts, query, tags } = req.body ?? {};
+
       // Create new dashboard from name and charts
       const newDashboard = await new Dashboard({
         name,
         charts,
         query,
-        tags,
+        tags: tags && uniq(tags),
```

Comment:
double check that we want to assign this undefined instead empty array ?

Reply:
yes, just want to skip updating this field if it's not passed. just in case we run old app and new api at the same time, to prevent data loss